server/shadow/shadow_client.c

```diff
@@ -817,85 +817,6 @@ int shadow_client_surface_update(rdpShadowClient* client, REGION16* region)
 	return 1;
 }
 
-int shadow_client_convert_alpha_pointer_data(BYTE* pixels, BOOL premultiplied,
```

Comment:
I actually hesitated to keep it private for X11 or common for all. Your suggestion give me additional reason to make the decision :)